src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,44 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
```

Comment:
For pass tests and do destructure you should use curly brackets after arrow or use destructure inside params 
```suggestion
export const MovieCard = ({ movie: { imgUrl,  title, description, imdbUrl } }) => (
```